requirements.txt

```diff
@@ -52,3 +52,6 @@ django-rest-swagger==0.3.2
 djangorestframework-bulk==0.2.1
 pyjwt==1.4.0
 pyjwe==0.1.6
+
+# Admin requirements
+django-extensions==1.5.9
```

Comment:
I think this was used for debugging urls. I think it could be strictly dev or removed for now. It does give you some model mixing but I don't think we're using any of those in this PR.